index.d.ts

```diff
@@ -44,6 +44,21 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+/**
+ * Hook that calls inputHandler callback with input that program received.
+ * Additionally contains helpful metadata for detecting when arrow keys were pressed.
+ */
+export function useInput(
+	inputHandler: (input: string, meta: Meta) => void
+): void;
+
+export interface Meta {
```

Comment:
I don't think `Meta` is clear enough as an export name. I would name it `MetaKey`.